src/Sarif.TestUtilities/AssertionExtensions.cs

```diff
@@ -17,17 +17,12 @@ public static class AssertionExtensions
         {
             Execute.Assertion
                 .ForCondition(
-                    string.Equals(RemoveLineEndings(expected), RemoveLineEndings(assertion.Subject), StringComparison.OrdinalIgnoreCase)
+                    FileDiffingTests.AreEquivalentSarifLogs<SarifLog>(actualSarif: assertion.Subject, expected)
```

Comment:
>AreEquivalentSarifLogs [](start = 37, length = 22)

This helper is more resilient to whitespace-only differences in test files. Also DRYs things out a bit.